wigwam/_utils.py

```diff
@@ -203,6 +203,24 @@ def test_image(image: Image, expression: str) -> bool:
             raise
 
 
+def get_libdir(
+    base_tag: str,
+) -> str:
+    """Determine if the given image uses `lib64` or `lib` as its `LIBDIR`."""
+    # Unlike with the CMake Install step, `libdir` can be checked directly on this image
+    # because it has something at $INSTALL_PREFIX. Check the base tag for lib64 or lib.
+    with temp_image(base_tag) as temp_img:
+        uses_lib64 = test_image(image=temp_img, expression='"$INSTALL_PREFIX/lib64"')
+
+    if uses_lib64:
+        libdir: str = "lib64"
+    else:
+        libdir = "lib"
+        assert test_image(image=temp_img, expression='"$INSTALL_PREFIX/lib"')
```

Comment:
`assert` probably isn't the right thing to do here anymore. Assertions are supposed to check your (the programmer's) own logic -- they shouldn't be tripped by users.

Since this is now a public function and the user can specify the input image, it's now possible for users to trip this assertion.

Let's change this to instead raise a `ValueError` or `RuntimeError` if neither directory (`lib` or `lib64` existed). Maybe something like this:

```suggestion
    with temp_image(base_tag) as temp_img:
        for libdir in ["lib64", "lib"]:
            if test_image(temp_img, f'"$INSTALL_PREFIX/{libdir}"'):
                return libdir
        else:
            raise ValueError(
                "could not find a directory named $INSTALL_PREFIX/lib64"
                " or $INSTALL_PREFIX/lib in the specified image"
            )
```